src/org/gh/RunMe.java

```diff
@@ -12,28 +12,34 @@ public static void main(String args[]){
         System.out.println("Creating a new student and Teacher");
         Student objStudent = new Student("Student 1", "GradeA");
         Teacher objTeacher = new Teacher("Teacher 1", "Class#1, Class#2");
-
+        McdWork objMcdWork = new McdWork ("Worker 1"), "Position, Salary"); 
```

Comment:
On this line, the parameters should be new McdWork("NAME", "GRADE"); (See the constructor in McdWork.java

But you've got an extra parameter (position and salary), and you don't have a Grade.

Also, you have an extra ")" before the comma after "Worker 1"